apps/bfd-pipeline/bfd-pipeline-shared-utils/src/main/java/gov/cms/bfd/pipeline/sharedutils/SamhsaUtil.java

```diff
@@ -96,19 +127,56 @@ public void createSamhsaMap() throws IOException {
    * @param claim The claim to process.
    * @param entityManager the EntityManager used to persist the tag.
    * @param <TClaim> Generic type of the claim.
+   * @return true if a tag was persisted.
    */
-  public <TClaim> void processClaim(TClaim claim, EntityManager entityManager) {
-    switch (claim) {
-      case RdaFissClaim fissClaim -> {
-        Optional<List<FissTag>> tags = Optional.of(checkAndProcessFissClaim(fissClaim));
-        persistTags(tags, entityManager);
-      }
-      case RdaMcsClaim mcsClaim -> {
-        Optional<List<McsTag>> tags = Optional.of(checkAndProcessMcsClaim(mcsClaim));
-        persistTags(tags, entityManager);
+  public <TClaim> boolean processClaim(TClaim claim, EntityManager entityManager) {
```

Comment:
Do we need the return value here? Doesn't seem like it's used right now.

Reply:
It will be used in the backfill code, to keep track of how many tags were created.